src/calc/calculate_phaseMerge.py

```diff
@@ -35,6 +35,10 @@ def calculate(self, reward_logs):
                 total_current_balance = sum([rl.current_balance for rl in rl_list])
                 total_ratio = sum([rl.ratio for rl in rl_list])
                 total_payment_amount = sum([rl.amount for rl in rl_list])
+                total_adjusted_payment_amount = sum(
```

Comment:
you seem to be replacing amount and payment amount by an entry called adjusted amount, this seemed focus by the adjustment mode logic, but does not fit the logic when amounts are not in adjusted mode.
IMO adjusted label should be to specific adjusted mode code, if at we are at -5 and not -11 it makes no sense when parsing the code.